src/pk_core.c

```diff
@@ -906,6 +906,66 @@ DEF(stdMathTangent,
   RET(VAR_NUM(tan(rad)));
 }
 
+DEF(stdMathSinh,
+  "sinh(rad:num) -> num\n"
+  "Return the hyperbolic sine value of the argument [rad] which is an angle "
+  "expressed in radians.") {
```

Comment:
The parameter isn't an angle, just the number, This should be `Return the hyperbolic sine value of the argument [val]`.